pkg/keystone/schema.go

```diff
@@ -0,0 +1,35 @@
+package keystone
```

Comment:
We're going to have to create this exact same package everywhere we use it. I wouldn't be a fan of that, and hence using Unstructured for this purpose.

However, I have also been wondering if we should rename the Keystone operator to the ControlPlane operator, and add all the control plane CRDs, with their own controllers, to the same operator. We could move it to lib-common, but that would also suck more than just importing it from mariadb-operator directly.

Reply:
We do not want this type of code in lib-common as it would significantly add to the set of goland dependencies we have to keep in lockstep. I think the idea with go libraries is keep dependencies simple. What we could do is have a lib-copy where we have tests on common code that we (manually) copy into some of our operators.

With regards to renaming KeystoneOperator I would ask this: Is there any chance someone might want to use parts of our Ctlplane elsewhere? Also putting all apps into the same repos seems to go against best practices (https://github.com/operator-framework/community-operators/blob/master/docs/best-practices.md) "If an application consists of multiple different tiers or components, multiple Operators should be written for each of them. If the application for example consists of Redis, AMQ and MySQL, there should be 3 Operators, not one."

Reply:
The lib-copy idea I mentioned above is in fact how oslo in OpenStack first started out btw. (we copied things around). It wasn't great, but that could provide re-use albeit via periodic (manual) copying of the libs around. And we could have a CI job alert of us things are out of sync.